docker/jenkins-images/Dockerfile-k8s-test

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:latest
+
+RUN apt-get update \
+	&& apt-get install curl apt-utils unzip make jq gettext-base -y
```

Comment:
Nitpick: good practice is to cleanup apt caches after apt-get (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)

Reply:
Thank you. Added to both dockerfiles after all the scripts are done.

`RUN rm /var/lib/apt/lists -rf`.'

Commit : https://github.com/h2oai/h2o-3/pull/4439/commits/88b61966a66850243c5c0a1f3778a8423883ddb9